src/test/java/com/julienvey/trello/integration/OrganizationGetITCase.java

```diff
@@ -42,6 +43,14 @@ public void setUp() {
         trello = new TrelloImpl(TEST_APPLICATION_KEY, "", httpClient);
     }
 
+    @Test
+    public void testGetOrganization() {
+        Organization organization = trello.getOrganization(ORGANIZATION_ID);
+
+        assertThat(organization).isNotNull();
+        assertThat(organization.getId()).isEqualTo(ORGANIZATION_ID);
+    }
+
```

Comment:
have you ran all integration tests in your branch?

Reply:
All the tests are passed